templates/repo/pulse.tmpl

```diff
@@ -1,7 +1,7 @@
 <h2 class="ui header activity-header">
 	<span>{{DateUtils.AbsoluteLong .DateFrom}} - {{DateUtils.AbsoluteLong .DateUntil}}</span>
 	<!-- Period -->
-	<div class="ui floating dropdown jump">
+	<div class="ui floating dropdown jump flex-text-block">
```

Comment:
Why? "dropdown" already has "display: inline-flex"

Reply:
What you need to do is to add the "ui dropdown" selector here:

![image](https://github.com/user-attachments/assets/4a09ddd8-6f7d-4a60-b194-d5bbbf033717)